openfl-tutorials/interactive_api/HPU/PyTorch_TinyImageNet/README.md

```diff
@@ -0,0 +1,140 @@
+# PyTorch_TinyImageNet
```

Comment:
General: Ensure consistency across your markdown for sections, subsections, lists, code snippets etc.
Use [this](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as a good starting point.

Reply:
Updated